js/src/components/paid-ads/campaign-preview/index.js

```diff
@@ -0,0 +1,13 @@
+export { default } from './campaign-preview';
+
+/**
+ * Data for compositing ad preview mockups.
+ *
+ * @typedef {Object} AdPreviewData
+ * @property {string} title Price of the product.
```

Comment:
❓ Should this be "Title of the product"?

Reply:
Thanks for catching this. Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1650/commits/8e72981c45e5ba8bc62f09bb2c1bb8bf1ebb7fbb.